core/signer/signer.go

```diff
@@ -93,11 +93,6 @@ func (s *Signer) StartSigning(interval time.Duration) {
 
 // ProcessMutation saves a mutation and adds it to the append-only log and tree.
 func (s *Signer) ProcessMutation(ctx context.Context, txn transaction.Txn, index, mutation []byte) error {
-	// Send mutation to append-only log.
```

Comment:
Consider whether it might be possible to reuse or extend the appender library by just a bit? Might save some work.